services/route_manager/src/test/java/com/futurewei/alcor/route/VpcRouterTests.java

```diff
@@ -334,7 +331,7 @@ public void updateSubnetRouteTable_pass () throws Exception {
         Mockito.when(routeTableDatabaseService.getAllRouteTables(anyMap()))
                 .thenReturn(new HashMap<String, RouteTable>(){{put(UnitTestConfig.routeTableId, routetable);}});
         Mockito.when(neutronRouterService.updateRoutingRule(anyString(), any(NewRoutesWebRequest.class), anyBoolean()))
-                .thenReturn(new UpdateRoutingRuleResponse(){{setHostRouteToSubnet(new ArrayList<>());}});
+                .thenReturn(new UpdateRoutingRuleResponse(){{setHostRouteToSubnet(new ArrayList<>());setInternalSubnetRoutingTable(new InternalSubnetRoutingTable(){{setRoutingRules(new ArrayList<>());}});}});
```

Comment:
";set" => what this line is about?

Reply:
The Set method here is because in the code, there is some logic to verify whether the value of these properties is null. If the value is null, an error will be reported. Therefore, we need to set the value of the properties in UT to ensure that they are not null

Reply:
Cool! Glad to see you back 👍

Reply:
Happy to contribute myself to the community again!